src/components/Footer/FooterLogoAndStatus.tsx

```diff
@@ -0,0 +1,17 @@
+import Logo from '@ably/ui/core/Logo';
+import FeaturedLink from '@ably/ui/core/FeaturedLink';
+import React from 'react';
+import { DOCUMENTATION_PATH } from '../../../data/transform/constants';
+
+export const FooterLogoAndStatus = () => (
```

Comment:
This is an odd combination of things; is there a reason these two are tied together?
If not, I would suggest making FooterStatus its own component and just rendering <Logo> and <FooterStatus> alongside each other
If there is a good reason, you should use that reason as the name of this component